### PR TITLE
fix(orc8r): move constants from doc.go files

### PR DESCRIPTION
**Pull request by spikey979**
fix(orc8r): move constants from doc.go files

## Summary

Moved constants from doc.go files to a corresponding const.go files

For issue:
https://github.com/magma/magma/issues/9828

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

File: orc8r/cloud/go/services/streamer/const.go [hunk 1]
```diff
@@ -16,5 +16,6 @@ package streamer
 import "syscall"
 
 const (
-	EAGAIN = syscall.EAGAIN
+	EAGAIN      = syscall.EAGAIN
+	ServiceName = "streamer"
```
Comment (themarwhal) [inline orc8r/cloud/go/services/streamer/const.go:20]: Why is the casing different? it looks like it used to be STREAMER?

Comment (spikey979) [reply orc8r/cloud/go/services/streamer/const.go:20]: I solve this issue as well: https://github.com/magma/magma/issues/9829

Comment (spikey979) [reply orc8r/cloud/go/services/streamer/const.go:20]: Here is my PR for lowercasing: https://github.com/magma/magma/pull/10377

Comment (themarwhal) [reply orc8r/cloud/go/services/streamer/const.go:20]: ohhhh I see! Looks good then :)

**Review by hcgatewood — APPROVED** [review]
Thanks for the fix! I'd say wait till tomorrow, rebase onto latest master, then re-push to see if we can get the flaky checks to pass

**Review by hcgatewood — APPROVED** [review]

File: lte/gateway/c/core/oai/test/s6a_task/s6a_test.cpp [hunk 2]
```diff
@@ -21,4 +21,4 @@ int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
   OAILOG_INIT("S6A", OAILOG_LEVEL_DEBUG, MAX_LOG_PROTOS);
   return RUN_ALL_TESTS();
-}
```
Comment (themarwhal) [inline lte/gateway/c/core/oai/test/s6a_task/s6a_test.cpp]: we can remove this change :)

Comment (spikey979) [reply lte/gateway/c/core/oai/test/s6a_task/s6a_test.cpp]: I removed that change... sorry about that

**Pull request merged**